core/arch/arm/plat-nuvoton/platform_config.h

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2014-2023, Linaro Limited
+ * Copyright (C) 2022-2023 Nuvoton Ltd.
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
+#include <stdint.h>
+
+/* Make stacks aligned to data cache line length */
+#define STACK_ALIGNMENT		64
+#define DRAM0_BASE	0x00000000
+#define DRAM0_SIZE	0x40000000	/* 1G DDR */
+#define GIC_BASE	0xDFFF8000
+#define IT_UART1	38
+#define UART0_BASE	0xf0000000
+#define UART_REG_SIZE	0x100
+#define CONSOLE_UART_BASE	UART0_BASE
+#define CONSOLE_UART_CLK_IN_HZ	1
```

Comment:
`CONSOLE_UART_CLK_IN_HZ` stands for UART bus clock frequency (in Hz) of the console. It is defined by few platforms that embed a console driver that use it for its configuration. _ns16550.c_ driver your platform relies on (`CFG_16550_UART`) does not use this config, so prefer remove it.

Reply:
Done